testing/test_nodes.py

```diff
@@ -71,7 +71,7 @@ def test_subclassing_both_item_and_collector_deprecated(
         ),
     ):
 
-        class SoWrong(nodes.File, nodes.Item):
+        class SoWrong(nodes.Item, nodes.File):
```

Comment:
This is the order which triggers the crash before this patch, and seems to be more common, so I just switched the test to use it.

For reference, in the plugin corpus I couldn't find any which directly do File, Item, but found these which do Item, File:

```
pytest-translations/pytest_translations/mo_files.py
11:class MoFileItem(Item, File):

pytest-checkipdb/pytest_checkipdb.py
27:class CheckIpdbItem(pytest.Item, pytest.File):

pytest-todo/todo/plugin.py
22:class TodoItem(pytest.Item, pytest.File):

pytest-isort/pytest_isort/__init__.py
167:class IsortItem(pytest.Item, pytest.File):

pytest-reqs/pytest_reqs.py
122:class ReqsItem(pytest.Item, pytest.File):

pytest-mccabe/pytest_mccabe.py
60:class McCabeItem(pytest.Item, pytest.File):

pytest-yapf/pytest_yapf.py
27:class YapfItem(pytest.Item, pytest.File):

pytest-flake8/pytest_flake8.py
95:class Flake8Item(pytest.Item, pytest.File):

pytest-black/pytest_black.py
45:class BlackItem(pytest.Item, pytest.File):

pytest-eradicate/pytest_eradicate.py
48:class EradicateItem(pytest.Item, pytest.File):

pytest-cram/pytest_cram/__init__.py
50:class CramItem(pytest.Item, pytest.File):
```